server/data/curiousClient.test.ts

```diff
@@ -51,27 +52,28 @@ describe('curiousClient', () => {
           additionalLDDAndHealthProblems: [],
         },
       ]
-      curiousApi.get(`/learnerProfile/${prisonNumber}`).reply(200, prisonerProfile)
+      curiousApi.get(`/learnerProfile/${prisonNumber}?establishmentId=${establishmentId}`).reply(200, learnerProfile)
 
       // When
-      const actual = await curiousClient.getPrisonerProfile(prisonNumber, systemToken)
+      const actual = await curiousClient.getLearnerProfile(prisonNumber, establishmentId, systemToken)
 
       // Then
-      expect(actual).toEqual(prisonerProfile)
+      expect(actual).toEqual(learnerProfile.pop())
```

Comment:
Ref: my comment below in the implementation re: popping from the array; I wonder if we need to make the test clearer - perhaps make the stub return an array of several elements, and then be explicit about which one we expect?